sdk/iot/Azure.Iot.Hub.Service/src/API Design.md

```diff
@@ -107,7 +107,65 @@ APIs for managing configurations for devices and modules.
 <details>
 
 ```csharp
+/// <summary>
+/// Gets a configuration on the IoT Hub for automatic device/module management
```

Comment:
do we call this "... on the IoT Hub..." or "... on the IoT Hub _instance_..." in our other comments?

Reply:
other comments as in for other configurations APIs?